pages/audits.js

```diff
@@ -37,12 +38,12 @@ export default function AuditDoc({ auditDoc }) {
         );
         if (filteredAudits.length > 0) {
           return (
-            <>
-              <h2 className="mb-1 mt-8 text-lg font-semibold text-brand">
+            <Fragment key={keyedAudits}>
```

Comment:
The key has to be a number or string, but `keyedAudits` is an array, so I see lots of big errors in the browser console now. Use `itemType` which is a string that’s unique for every iteration.

Reply:
oh sorry. I thought i had changed it following our conversation. Sorry about that!